modules/combined/test/tests/cavity_pressure/tests

```diff
@@ -1,64 +1,86 @@
 [Tests]
   issues = '#2703'
   design = 'CavityPressure/index.md'
-  [./test]
-    type = 'Exodiff'
-    input = 'cavity_pressure.i'
-    exodiff = 'cavity_pressure_out.e'
-    abs_zero = 1e-10
-    rel_err = 1e-5
+  [values]
     requirement = 'The system shall compute a cavity pressure based on the ideal gas law'
-  [../]
+    [./3d]
+      type = 'Exodiff'
+      input = '3d.i'
+      exodiff = '3d_out.e'
+      abs_zero = 1e-10
+      rel_err = 1e-5
+      detail = 'in 3D.'
+    [../]
+    [./initial_temperature]
+      type = 'Exodiff'
+      input = 'initial_temperature.i'
+      exodiff = 'initial_temperature_out.e'
+      abs_zero = 1e-10
+      rel_err = 1e-5
+      requirement = 'where the initial temperature of the cavity is independently specified.'
```

Comment:
@aeslaughter - This should have failed the doc build. 

@tophmatthews - When doing nesting, you can't have another requirement. There should be one requirement at the group level and then "detail = <short detail>" that differentiates this requirement from the others in the group. Thanks for the sanity check on the documentation build 😉.

Reply:
ah whoops, forgot to change it to detail